lib/membrane/video_compositor/object/input_image.ex

```diff
@@ -1,7 +1,9 @@
-defmodule Membrane.VideoCompositor.Scene.Object.Input.StaticFrame do
+defmodule Membrane.VideoCompositor.Object.InputImage do
```

Comment:
I think I like `Image` and `Video` more than `InputImage` and `InputVideo`.
What do you think?

Reply:
I actually think it's fine as it is